src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -6,7 +6,10 @@
 using System.Runtime.InteropServices;
 using System.Runtime.Intrinsics.X86;
 
+#if CORECLR
+// Downstream will use System.Runtime.CompilerServices
 using Internal.Runtime.CompilerServices;
+#endif
```

Comment:
Not sure if this or similar will help with the `CoreFX` sync [crazy](https://github.com/dotnet/corefx/pull/35340)